src/decode.c

```diff
@@ -382,6 +383,20 @@ void PacketDefragPktSetupParent(Packet *parent)
     DecodeSetNoPayloadInspectionFlag(parent);
 }
 
+int PacketHasBypassCallback(Packet *p)
```

Comment:
why do you need this check function? Seems easier and more efficient to just call PacketBypassCallback and do the check there.